src/vs/workbench/api/browser/mainThreadExtensionService.ts

```diff
@@ -27,7 +27,7 @@ import { IExtensionHostProxy, IResolveAuthorityResult } from 'vs/workbench/servi
 import { ActivationKind, ExtensionActivationReason, IExtensionService, IInternalExtensionService, MissingExtensionDependency } from 'vs/workbench/services/extensions/common/extensions';
 import { extHostNamedCustomer, IExtHostContext, IInternalExtHostContext } from 'vs/workbench/services/extensions/common/extHostCustomers';
 import { Dto } from 'vs/workbench/services/extensions/common/proxyIdentifier';
-import { IHostService } from 'vs/workbench/services/host/browser/host';
+import { IHostService } from 'vs/workbench/services/host/common/host';
```

Comment:
@bpasero I'd like to reference `IHostService` from `common` and couldn't do that because the interface is defined in `browser`. Based on the jsdoc for the interface it looks like it could just as well be in `common`, do you have any concerns about moving it to `common`?

Reply:
No strong objections, but the reason I have it in `browser` is because its never going to be possible to implement the interface from `common` as it requires some kind of UI-host, either `browser` or `electron-sandbox`.

Reply:
What is the new "thing" that requires the host service to be in `common`?

Reply:
I'd like to use `IHostService#openWindow` to open a workspace folder is all, all other usages are from `browser`

Reply:
@joyceerhl would it not be easier to move `chatServiceImpl` into `browser` then?